openfecwebapp/templates/advanced.html

```diff
@@ -2,18 +2,18 @@
 {% import 'macros/page-header.html' as header %}
 
 {% block title %}
-  Advanced data
+  {{ title }}
 {% endblock %}
 
 {% block body %}
-  {{ header.header('Advanced data', '', show_search=False) }}
+  {{ header.header(title, '', show_search=False) }}
   <div class="main">
   <div class="container">
-    <h1 class="icon-heading--circle-table">Advanced data index</h1>
+    <h1 class="icon-heading--circle-table">{{ title }}</h1>
```

Comment:
Yep, that was the intention. @emileighoutlaw noted the desired change [here](https://github.com/18F/openFEC-web-app/issues/1245#issuecomment-223981578)